BrushedMotorControllerFirmware/include/uart_shell/shell_direct_motor_control.c

```diff
@@ -93,6 +93,22 @@ static int cmd_position(const struct shell *shell, size_t argc, char *argv[])
 	return 0;
 }
 
+static int cmd_position_zero(const struct shell *shell, size_t argc, char *argv[])
+{
+	return_codes_t ret;
+
+	ret = position_reset_zero(get_relevant_channel());
+
+	if (ret == SUCCESS) {
+		shell_fprintf(shell, SHELL_NORMAL, "Position reset succesfulll\n");
```

Comment:
```suggestion
		shell_fprintf(shell, SHELL_NORMAL, "Position reset succesfull\n");
```

Reply:
Already covered in https://github.com/eSqadron/NRF52840-Brushed-Motor-Controller-Firmware/pull/11